lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
If you go down this route, here's a suggested interface: replace the `correlationIds` param with an `additionalProperties` param of type `{ [index: string]: any }` which gets written to the output. Either flatten them into the top level or, if you want to not have any potential collisions, write it as-is.